routes/home.js

```diff
@@ -7,12 +7,17 @@ const bodyParser = require("body-parser");
 const jsonParser = bodyParser.json();
 
 const {home, getAboutPage, getAllBreaches, getBentoStrings, getSecurityTips, protectMyEmail, addEmailToRelayWaitlist, notFound} = require("../controllers/home");
-const { requireSessionUser } = require("../middleware");
+const { requireSessionUser, getShareUTMs } = require("../middleware");
 
 const router = express.Router();
 const csrfProtection = csrf();
 
 router.get("/", csrfProtection, home);
+router.get("/share/orange", csrfProtection, getShareUTMs, home);
```

Comment:
Added middleware function to catch `/share/` link requests and set the UTMs accordingly